automated-actions/AWS_RISK_CREDENTIALS_EXPOSED/cloudformation/risk_credentials_exposed.serverless.yaml

```diff
@@ -0,0 +1,252 @@
+      EventPattern:
+        source: 
+          - "aws.health"
+        detail-type: 
+          - "Trusted Advisor Check Item Refresh Notification"
+        detail: 
+          service: 
+            - "RISK"
+          eventTypeCategory:
+            - "issue"
+          eventTypeCode:
+            - "AWS_RISK_CREDENTIALS_EXPOSED"
+      State: "ENABLED"
+      Targets: 
+        - 
+          Arn: !Ref ExposedKeyStepFunction
+          Id: "TargetFunctionV1"
+          RoleArn: !GetAtt ExecuteStateMachineRole.Arn
+
+  ExecuteStateMachineRole:
+    Type: "AWS::IAM::Role"
+    Properties:
+      AssumeRolePolicyDocument:
+        Version: "2012-10-17"
+        Statement:
+          -
+            Sid: "AllowCWEServiceToAssumeRole"
+            Effect: "Allow"
+            Action:
+              - "sts:AssumeRole"
+            Principal:
+              Service:
+                - "events.amazonaws.com"
+      Path: "/"
+      Policies:
+        -
+          PolicyName: "ExecuteStateMachine"
+          PolicyDocument:
+            Version: "2012-10-17"
+            Statement:
+              -
+                Effect: "Allow"
+                Action:
+                  - "states:StartExecution"
+                Resource: "*"
+
+  ExposedKeyStepFunction:
+    Type: AWS::StepFunctions::StateMachine
+    Properties:
+      DefinitionString: !Sub |-
+        {
+          "Comment": "Deletes exposed IAM access keypairs and notifies security",
+          "StartAt": "DeleteAccessKeyPair",
+          "States": {
+            "DeleteAccessKeyPair": {
+              "Type": "Task",
+              "Resource": "${DeleteAccessKeyPair.Arn}",
+              "Next": "LookupCloudTrailEvents"
+            },
+            "LookupCloudTrailEvents": {
+              "Type": "Task",
+              "Resource": "${LookupCloudTrailEvents.Arn}",
+              "Next": "NotifySecurity"
+            },
+            "NotifySecurity": {
+              "Type": "Task",
+              "Resource": "${NotifySecurity.Arn}",
+              "End": true
+            }
+          }
+        }
+      RoleArn: !GetAtt StepFunctionExecutionRole.Arn
+
+  StepFunctionExecutionRole:
+    Type: "AWS::IAM::Role"
+    Properties:
+      AssumeRolePolicyDocument:
+        Version: "2012-10-17"
+        Statement:
+          - Effect: "Allow"
+            Principal:
+              Service: !Sub |-
+                states.${AWS::Region}.amazonaws.com
+            Action: "sts:AssumeRole"
+      Path: "/"
+      Policies:
+        - PolicyName: StatesExecutionPolicy
+          PolicyDocument:
+            Version: "2012-10-17"
+            Statement:
+              - Effect: Allow
+                Action:
+                  - "lambda:InvokeFunction"
+                Resource: "*"
+
+  DeleteAccessKeyPair:
+    Type: AWS::Serverless::Function
+    Properties:
+      Handler: delete_access_key_pair.lambda_handler
+      Runtime: python3.6
+      CodeUri: ../lambda_functions/
```

Comment:
The risk_credentials_exposed.serverless.yaml file is a template meant to be built before use with the instructions detailed in the README, so this is currently as intended.